pkg/forwarder/telemetry.go

```diff
@@ -9,20 +9,12 @@ import (
 	"expvar"
 
 	"github.com/DataDog/datadog-agent/pkg/forwarder/transaction"
+	"github.com/DataDog/datadog-agent/pkg/orchestrator"
 	"github.com/DataDog/datadog-agent/pkg/telemetry"
 )
 
 var (
-	transactionsIntakePod         = expvar.Int{}
-	transactionsIntakeDeployment  = expvar.Int{}
-	transactionsIntakeReplicaSet  = expvar.Int{}
-	transactionsIntakeService     = expvar.Int{}
-	transactionsIntakeNode        = expvar.Int{}
-	transactionsIntakeJob         = expvar.Int{}
-	transactionsIntakeCronJob     = expvar.Int{}
-	transactionsIntakeCluster     = expvar.Int{}
-	transactionsIntakeDaemonSet   = expvar.Int{}
-	transactionsIntakeStatefulSet = expvar.Int{}
+	TransactionsIntakeOrchestrator = map[orchestrator.NodeType]*expvar.Int{}
```

Comment:
Potentially I could kill the import altogether and just use the `int`, with this I wouldn't need to move the `pod` file out of `orchestrator`. We can discuss this. I do like using explicit types here